### DISCUSSION TITLE
Error messages in VS Code when doing npm i

### DISCUSSION THREAD

**Discussion by DavevonH** [neuefische/questions#739] [Web] [answered]
Ich habe ein Problem in VS Code. Beim "npm i" Befehl werden mir folgende Meldungen angezeigt. Kann hier jemand Abhilfe schaffen?

![Bildschirmfoto 2023-09-26 um 12 06 56](https://github.com/neuefische/questions/assets/140090722/764e8223-b792-4336-934d-81aa4b9147de)

**Comment by moonwave99** [comment, marked as answer]
Hey David,

it looks like somebody installed the `run-db` package accidentally!
That package has `better-sqlite-3` under its deps, that's why you are getting this error.

How did I find it out? I opened the `package-lock.json` file, and searched for the problematic package until I found its parent ^^

Solution:

```
npm uninstall run-db
```

Don't forget to commit and push the changes to `package.json` after that!
